src/test/java/org/springframework/data/querydsl/WrapperToWrapWrapperForUserWrapper.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.querydsl;
+
+import com.mysema.query.annotations.QueryEntity;
+import com.mysema.query.annotations.QueryInit;
+
+/**
+ * @author Christoph Strobl
+ */
+@QueryEntity
+public class WrapperToWrapWrapperForUserWrapper {
+
+	@QueryInit("wrapper.user")//
```

Comment:
yes and no, spring data code formatter would otherwise move everything into one line